datanode/sqlstore/migrations/0012_candles_add_notional.sql

```diff
@@ -14,14 +17,14 @@ SELECT market_id, time_bucket('1 minute', synthetic_time) AS period_start,
 FROM trades
 GROUP BY market_id, period_start WITH NO DATA;
 
-DROP MATERIALIZED VIEW IF EXISTS trades_candle_1_minute;
-ALTER MATERIALIZED VIEW trades_candle_1_minute_tmp RENAME TO trades_candle_1_minute;
-
 -- start_offset is set to a day, as data is append only this does not impact the processing time and ensures
 -- that the CAGG data will be correct on recovery in the event of a transient outage ( < 1 day )
 SELECT add_continuous_aggregate_policy('trades_candle_1_minute', start_offset => INTERVAL '1 day', end_offset => INTERVAL '1 minute', schedule_interval => INTERVAL '1 minute');
```

Comment:
For each view, before reenabling the continuous aggregate, I think the following call will achieve this, but test it as not 100%.

refresh_continuous_aggregate("viewname", null, null)